.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--
+Please make sure you've read and understood our contributing guidelines;
+https://github.com/Azure/SONiC/blob/gh-pages/CONTRIBUTING.md
+
+Please provide following information to help code review process a bit easier:
+-->
+### Description of PR
+<!--
+- Please include a summary of the change and which issue is fixed.
+- Please also include relevant motivation and context. Where should reviewer start? background contaxt?
+- List any dependencies that are required for this change.
+-->
+Fixes # (issue)
+
+### Type of change
+- [ ] Bug fix
+- [ ] Testbed and Framework(new/improvement)
+- [ ] Test case(new/improvement)
+
+### Approach
+How did you do it?
```

Comment:
if it is new test case, we need them to provide which testbed the test will run.